### DISCUSSION TITLE
"WARNING  NameOfMyWidget (pseudo_classes={'enabled'}) IS NOT RUNNING, Prompt() not sent" and the like - should I be worried?

### DISCUSSION THREAD

**Discussion by MaineTim** [Textualize/textual#2023] [Q&A] [answered]
So we have a new warning message in 0.14, and I'm getting a bunch of them on existing code.  I can see from the code in widget.py that generates them that there's been an attempt to post a message to an object that's no longer there, but I don't know enough to know if I need to worry about it. It kinda looks like it's just trying to make me aware of a condition I might need to know about in certain circumstances, and not in many others, but the big red "WARNING" sends a more ominous tone. So, without posting example code, in a generic sense, is this just a friendly "you should just be aware" type of message, or is it  the "Beware: clean up your act now!" kind? If it's the latter, then I can pursue figuring out why I'm behaving so badly.

Thanks!

**Comment by willmcgugan** [comment, marked as answer]
You can get those warnings when things are working normally. Unless you get them consistently for the same widget, there is probably nothing to fix.

We will refine those warnings in a future version.

**Reply by MaineTim** [reply]
Thank you!
